src/app/project/board/dataset-metadata.ts

```diff
@@ -0,0 +1,123 @@
+/* Interface for url type */
```

Comment:
I would remove all comments for interfaces which are self-descriptive and leave only these which are not.

Reply:
Done